### ISSUE TITLE
"Page not found" hardcoded in NotFoundPage

### ISSUE THREAD

**Issue by reymus** [gpbl/isomorphic500#70]
The message "Page not found" is hardcoded in the NotFoundPage. Problem is easily reproducible by switching to French and clicking the "an unexisting photo" link in the footer.

**Comment by reymus** [comment]
Pull request: https://github.com/gpbl/isomorphic500/pull/71

**Comment by Wykks** [comment]
Well, to be consistent, the message in ErrorPage should be translated too.

**Comment by gpbl** [comment]
Thanks... there are other texts missing, yes: I actually meant to provide an example for i18n, not to translate all the things :-) otherwise the project get too complex to maintain.

**Comment by Wykks** [comment]
Yeah that's why I didn't done it first.This is just about *consistency* because having the NotFoundPage translated but not the ErrorPage isn't logical.
